src/components/LuxMenuBar.vue

```diff
@@ -473,7 +477,6 @@ export default {
       }
 
       .lux-show a {
-        &:hover,
```

Comment:
On hover in light theme we would want
&:hover {
text-decoration-color: var(--color-princeton-orange-on-white);
}